examples/plot_binary_classification_COMPAS.py

```diff
@@ -0,0 +1,486 @@
+"""
+=======================================
+Binary Classification on COMPAS dataset
+=======================================
+"""
+print(__doc__)
```

Comment:
Was this supposed to show up on the rendered webpage?

Reply:
yes. It's there to show up when you run the `.py` version of the example.

Reply:
What does it print? All of the (former) notebook text?

Reply:
Yes. Could remove it if you want.

Reply:
Maybe leave it in for now; I think it looks a little odd, but I don't have the only vote here :-)

When you push your changes, I'll bring them over to my branch, and get the updated sample documentation.